srcDoc/release_notes.rst

```diff
@@ -54,6 +54,13 @@ What's New in Master
 What's Fixed in Master
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Error using Micro with SHMEM when the test finalize
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An error using *RTI Perftest* for micro 3.0.0 has been solved by unregistering
+'_shmem' before finalize_instance().
```

Comment:
When using *RTI Connext DDS Micro* and setting the transport to `SHMEM` an error would appear at the end of the test in both publisher and subscriber by the time he `finalize_instance()` function is called. This erros has been resolved.